### PR TITLE
New provider: Elitetorrent

### PR DESCRIPTION
**Pull request by CristianBB**
New provider for spanish shows:
- RSS support for daily searches
- Backlog support
- Seeders available on results
- Leechers available on results
- Size don't returned on search results (#FIXME)

Please, take a look to _processTitle method:
- No generic quality strings (adapted to SR)
- "[Spanish Audio]" appended on show results, simple way to identify language of torrents returned by this provider (If we dont want them, use blacklist)

And _get_episode_search_strings method:
- Naming pattern: show 1x01 

Anything please let me know

### CODE REVIEW

File: sickbeard/providers/elitetorrent.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+            'cat': 4,
+            'modo': 'listado',
+            'orden': 'fecha',
+            'pag': 1,
+            'buscar': ''
+
+        }
+        
+    def _get_episode_search_strings(self, ep_obj, add_string=''):
+
```
Comment (miigotu) [inline sickbeard/providers/elitetorrent.py]: remove this method.

Comment (CristianBB) [inline sickbeard/providers/elitetorrent.py]: @miigotu In that case, how can I change the original search string ("show S01E01") to needed search string ("show 1x01")?. It appears the same method than original, the only change is:

sickbeard.config.naming_ep_type[2] 
to:
sickbeard.config.naming_ep_type[0] 

Maybe there is other way that I didn't see

Comment (miigotu) [inline sickbeard/providers/elitetorrent.py]: do an re.sub on the string inside _doSearch in that case.

something like:
`search_string = re.sub(r'S0*(\d*)E(\d*)', r'\1x\2', search_sring)`

Comment (CristianBB) [inline sickbeard/providers/elitetorrent.py]: Ok then, I thought that this was the best option cause I saw same method overwritten on others providers.
Let me go to eat and I'll make the changes.

Thanks @miigotu

**Comment by duramato** [comment]
@CristianBB Taking a look truh the site it seems the details page has size info..

**Comment by CristianBB** [comment]
@duramato You're right, but it appears inside details page so we would have to make a second getUrl to get that info (for every result returned...), If you want it of course I could implement it, but I think that it's not a good choice...

**Comment by CristianBB** [comment]
@miigotu method removed.
If you want, we can wait until @duramato says something about size parsing

**Comment by miigotu** [comment]
I'll merge it, but in the future you need to do:
`git diff --check upstream/develop` (or origin) to check for whitespace errors.

``` diff
sickbeard/providers/elitetorrent.py:52: trailing whitespace.
+        
sickbeard/providers/elitetorrent.py:68: trailing whitespace.
+        
sickbeard/providers/elitetorrent.py:75: trailing whitespace.
+        
sickbeard/providers/elitetorrent.py:78: trailing whitespace.
+            
sickbeard/providers/elitetorrent.py:87: trailing whitespace.
+                
sickbeard/providers/elitetorrent.py:90: trailing whitespace.
+                
sickbeard/providers/elitetorrent.py:93: trailing whitespace.
+                
sickbeard/providers/elitetorrent.py:95: trailing whitespace.
+                
sickbeard/providers/elitetorrent.py:115: trailing whitespace.
+                                leechers_raw = row.find('td', class_='clientes').text 
sickbeard/providers/elitetorrent.py:121: trailing whitespace.
+                                                                
sickbeard/providers/elitetorrent.py:153: trailing whitespace.
+   
sickbeard/providers/elitetorrent.py:160: trailing whitespace.
+            
sickbeard/providers/elitetorrent.py:165: trailing whitespace.
+                    
sickbeard/providers/elitetorrent.py:168: trailing whitespace.
+        
```

**Comment by CristianBB** [comment]
Pointed! Next time I'll remember it @miigotu

**Pull request merged**